app/src/main/java/com/gh4a/utils/StringUtils.java

```diff
@@ -139,4 +140,30 @@ public static SpannableStringBuilder applyBoldTags(Context context,
         }
         return ssb;
     }
+
+    @Nullable
+    public static int[] findMatchingLines(String input, String match) {
+        String[] lines = input.split("\n");
+        String[] matchLines = match.split("\n");
+
+        for (int i = 0; i < lines.length - matchLines.length; i++) {
```

Comment:
There's one issue in this loop: If input and match are exactly equal, the break condition in this line will cause no match to be found.